DEVELOPMENT.md

```diff
@@ -91,7 +90,13 @@ While iterating on the project, you may need to:
 $ go build ./cmd/...
 ```
 
-It builds `tkn` binary in your current directory. You can start playing with it.
+It builds `tkn` binary in your current directory. You can start playing with
+it. If you add the directory to your `$PATH` you can use your locally built
+binary from anywhere, e.g. you can add to your `.basrc` (or similar):
+
+```
+export PATH=$PATH:$GOPATH/src/github.com/tektoncd/cli/
```

Comment:
@vdemeester is this what ppl working on the project do? (I'm just guessing and interested in how other folks use a dev binary and keep it up to date!)

Reply:
I have my path set to `export PATH=~/bin:$PATH` and in `~/bin/` soft link to `~/go/src/path/to/tkn` binary. I know I could use `go install` but my method keeps `tkn` upto date all the time as there is a chance I miss `go install`

Reply:
ah kk that makes a lot of sense! im gonna tweak this a bit then

Reply:
I changed this a bit @sthaha based on what you described! What do you think?

Reply:
Thanks! that's indeed a lot nicer!